### ISSUE TITLE
Compatibility with screen readers on touch devices

### ISSUE THREAD

**Issue by simevidas** [WICG/inert#19]
I’ve made a demo to test the polyfill in VoiceOver on iPhone: https://output.jsbin.com/voxopus/quiet. When I swipe left/right to move to the previous/next item, it doesn’t skip over inert elements. To be honest, I’m not sure if it should. Someone with experience with screen readers on touch devices would know if swiping horizontally should skip inert elements.

**Comment by robdodson** [comment]
It's skipped over by TalkBack on Android. I'll check the example on my iPhone when I get into the office tomorrow. Because the `<p>` has aria-hidden and the underlying `<a>` has `tabindex=-1` it should be skipped over. If it's not being skipped over on iOS there may be an error in the polyfill on the current version of mobile safari

**Comment by alice** [comment]
Thanks for looking into this, @simevidas!

I just had a play, and I get the same results you do. I also tried setting `aria-hidden` explicitly immediately after `inert` was set and that does cause it to be correctly skipped by VoiceOver.

:thinking_face:

**Comment by robdodson** [comment]
Looks like setting the `inert` property isn't triggering anything. Setting it to `true` should poke the polyfill to set `aria-hidden=true` on the `p`

<img width="580" alt="screen shot 2016-08-07 at 6 57 35 pm" src="https://cloud.githubusercontent.com/assets/1066253/17467243/ebfbe842-5cd0-11e6-9cc9-047758fd6d8a.png">

**Comment by robdodson** [comment]
Issue might be just that we need to transpile the classes down to ES5
https://github.com/WICG/inert/blob/master/inert.js#L48

:P

**Comment by alice** [comment]
Yep that looks like it.

**Comment by robdodson** [comment]
Transpiled version may work https://output.jsbin.com/dageru
Gotta double check on a real phone tomorrow. My iOS simulator is outdated

**Comment by simevidas** [comment]
Works :+1:
